cdap-app-templates/cdap-etl/cdap-data-pipeline/src/main/java/co/cask/cdap/datapipeline/SmartWorkflow.java

```diff
@@ -177,6 +190,8 @@ public void initialize(WorkflowContext context) throws Exception {
       postActions.put(actionSpec.getName(), (PostAction) context.newPluginInstance(actionSpec.getName(),
                                                                                    macroEvaluator));
     }
+
+    WRAPPERLOGGER.info("Pipeline '{}' running", context.getApplicationSpecification().getName());
```

Comment:
Whether there is currently a mapping present for the states not relevant here. The concern is the assumption that the transition from started to running has happened regardless of the ProgramStatus [even though it might be true in today's implementation]. 
Since the PR is ready for merge otherwise, better to open a separate bug and discuss this over there.